src/grpc/dp_grpc_thread.cpp

```diff
@@ -29,7 +29,8 @@ static void *dp_grpc_main_loop(__rte_unused void *arg)
 		rte_exit(EXIT_FAILURE, "Cannot run without working gRPC server\n");
 
 	GRPCService::Cleanup();
-	return NULL;
+
```

Comment:
This should either be done for all functions here or none of them. It's meant to be a single "block" of state-change and return OK, like the other functions.

Reply:
removed.